features/networking/pod.feature

```diff
@@ -480,3 +480,39 @@ Feature: Pod related networking scenarios
     | ip route |
   Then the output should contain:
     | <%= cb.pod_cidr %> dev <%= cb.tunnel_inf_name %> |
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-26373
+  @admin
+  @destructive  
+  Scenario: Pod readiness check for OVN		
+  Given the env is using "OVNKubernetes" networkType
+  And OVN is functional on the cluster
+  Given I switch to cluster admin pseudo user
+  And I use the "openshift-ovn-kubernetes" project
+  And a pod is present with labels:
+    | l | app=ovnkube-node |
```

Comment:
It should be
```
  And a pod is present with labels:
    | app=ovnkube-node |
```

Reply:
Anyways it works both ways. Thanks Moved it to more optimised version as you suggested, Liang.